docs/howto/getting_started_markdown.md

```diff
@@ -0,0 +1,173 @@
+Getting Started Guide
+=====================
+
+Prerequisites
+-------------
+
+To get started with the Alteryx Core SDK, you need a valid installation
+of [Alteryx Designer](https://www.alteryx.com/).
+
+Python version 3.8.5 is required for Alteryx Designer plugins.
```

Comment:
TW review:

Update to bullet list, new URL for Alteryx Designer, also Q: is Alteryx Core SDK the name we should use? or should it by Alteryx Platform SDK?

- To get started with the Alteryx Core SDK, you need a valid installation of [Alteryx Designer](https://www.alteryx.com/products/alteryx-designer).
- Python version 3.8.5 is required for Alteryx Designer plugins.

Reply:
latest name is 'Alteryx Platform SDK', fixed and made a bulleted list